web-common/src/Component/Expand/Types.purs

```diff
@@ -0,0 +1,28 @@
+module Component.Expand.Types where
+
+import Halogen as H
+import Halogen.HTML as HH
+
+type Input ps pa m
```

Comment:
not inmidiatly obvious what `pa` and `ps` means. I like short letters for return values `a`, monads `m`, for the rest I prefer having the long boring but obvious meaning.

`ps` == `parentSlot`?
`pa` == `parentAction`?

Reply:
yeah, exactly. Sorry, probably a bad convention I was copying from community halogen libraries (e.g. Select and Formless)